RELEASE.md

```diff
@@ -1,88 +1,60 @@
-# Versioning
+## Changelog
 
-SRI releases follow [SemVer 2.0.0](https://semver.org/).
+The changelog is a file that contains a curated, chronologically ordered list of
+notable changes for each version of a project. It is a way to keep track of the
+project's progress and to communicate the changes to the users and the
+community. The changelog should be updated for each release and is part of the
+release process.
 
-Given a version number `MAJOR.MINOR.PATCH`, we increment the:
-- `MAJOR` version when incompatible API changes are introduced
-- `MINOR` version when functionality is added in a backward compatible manner
-- `PATCH` version when backward compatible bug fixes are introduced
+Things to take into account while writing a changelog:
 
-SRI has a global version, which uses git tags and keeps track of how the codebase evolves across time as a whole.
+ - General release information.
+ - Breaking changes.
+ - Notable changes.
 
-Every internal SRI crate also follows SemVer 2.0.0, but each crate version is only set on the respective `Cargo.toml`,
-(no git tags), and it evolves independently of the global release version.
 
-Whenever a `PATCH` is introduced, it is applied to all the latest `MAJOR` releases.
-For example: imagine there's releases `v1.0.0`, `v1.1.0`, and `v2.0.0`. A bug is found, dating back all the way to `v1.0.0`.
-A patch is applied such that the following new tags are introduced: `v1.1.1` and `v2.0.1`.
+## Release Process
 
-# Git Branching Strategy
+Try to constrain each development cycle to a fixed time period, after which a
+release is made. Typically, the development cycle is 6-8 weeks long.  After all
+release tasks are completed, initiate the release process by creating a new
+release branch, named `x.y.z.` referring to the version number, from the `main`
+branch.  The release branch is used as a breaking point and future reference for
+the release and should include the changelog entries for the release and any
+other release-specific tasks. Any bug fixes or changes for the release should be
+done on the release branch. When the release branch is ready, create a Pull
+Request from release branch to the `main` branch.  Once the Pull Request is
+approved and merged, create a new tag for the release branch and initiate any
+publishing tasks.
 
-We follow a simplified [gitflow](https://nvie.com/posts/a-successful-git-branching-model/) branching strategy.
 
-Although our strategy is very similar to the classic gitflow model, we do not keep release branches.
+Usually the release process is as follows:
 
-![](git-branching.png)
+1. Create a new release branch from the `main` branch.
+2. Amend the changelog with the new release information.
+3. Create a Pull Request from the release branch to the `main` branch.
```

Comment:
I don't understand this step. If the new release branch was just created from `main`, why do we want to merge it back to `main`?

Reply:
ok so the only file changed is the changelog file